build-system/tasks/e2e/package.json

```diff
@@ -7,8 +7,8 @@
     "@babel/register": "7.11.5",
     "babel-regenerator-runtime": "6.5.0",
     "chromedriver": "84.0.1",
```

Comment:
"chromedriver": "86.0.0",

Reply:
Just noticed that lol, wondering why yarn upgrade didn't do the trick